bundles/AdminBundle/Resources/public/js/pimcore/document/edit.js

```diff
@@ -166,7 +165,7 @@ pimcore.document.edit = Class.create({
                     mask.setStyle("opacity","0.5");
                     mask.setStyle("pointer-events","none");
                 } else {
-                    ed.unmask();
+                    editables[i].setInherited(editables[i].getInherited());
```

Comment:
Why have you changed that? 
Now masked items are not unmasked anymore, is this intentional?

Reply:
Actually ed.unmask() is removing inherited elements grey lock masks as well. So for unmasking items, instead of using ed.mask(), we can use setInherited method() to bring editables back to their state they were before user clicks "highlight editable elements". I tested it for both documents having master document and not having master document.